console/other-usefull-commands.rst

```diff
@@ -21,6 +21,27 @@ This comes in handy if you e.g. added a new state that you need to translate for
  Translation.create_if_not_exists( :locale => 'de-de', :source => "New", :target => "Neu", format: 'string', created_by_id: 1, updated_by_id: 1 )
 
 
+Translating Attributes
+~~~~~~~~~~~~~~~~~~~~~~
+
+By default Zammad will not translate Attributes. A small exception here is the state dropdown, which has translation enabled by default.
+With the following code you can enable translation, this will translate the attribute display name and the display names of values (if it's a value field).
```

Comment:
`With the following code you can enable translation, this will translate the attribute display name and the display names of values (if it's a value field).`

`With the following code you can enable translation. This will translate the attribute display name and the display names of values (if it's a value field).`

Reply:
Done with commit https://github.com/zammad/zammad-documentation/pull/95/commits/c5e8fc1d477a943a42f4f53327a5e4296584e2b7